pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: myWitWallet allows you to send and receive Wit immediately. Bye bye
 
 publish_to: 'none'
 
-version: 0.0.9-dev
+version: 0.0.0+0
```

Comment:
How versioning is going to work from now on? Are we using the `+` to specify [build metadata](https://semver.org/#spec-item-10)? How are we going to differenciate `production` and`internal team`?